src/sentry/testutils/helpers/notifications.py

```diff
@@ -89,3 +94,109 @@ def get_title_link(self, *args):
     "info",
     "/api/123/",
 )
+
+SAMPLE_TO_OCCURRENCE_MAP = {
```

Comment:
ended up manually creating occurrences for the transaction samples since I wasn't sure how to pull out occurrences out of save_transaction_event

Reply:
Yeah it's a bit of a pain because realistically `save_transaction_event` just produces to the kafka topic and we don't know when that will finish.

For tests, if you want you can do something similar to what I do in `test_adapter`

```
        where_conditions = [
            Condition(Column("project_id"), Op.EQ, self.project.id),
            Condition(Column("event_id"), Op.EQ, event.event_id),
            Condition(Column("timestamp"), Op.GTE, before_now(hours=1)),
            Condition(Column("timestamp"), Op.LT, datetime.now()),
        ]

        snuba_request = Request(
            dataset="search_issues",
            app_id="tagstore",
            query=Query(
                match=Entity("search_issues"),
                select=[
                    Column("group_id"),
                    Column("occurrence_id"),
                    Column("group_id"),
                ],
                where=where_conditions,
            ),
        )
        result = raw_snql_query(
            snuba_request, referrer="tagstore.get_generic_group_list_tag_value", use_cache=True
        )["data"][0]
        event = event.for_group(Group.objects.get(id=int(result["group_id"])))
        event.occurrence = IssueOccurrence.fetch(result["occurrence_id"], self.project.id)
        ```